packages/crystal/build.sh

```diff
@@ -0,0 +1,49 @@
+TERMUX_PKG_HOMEPAGE=https://crystal-lang.org
+TERMUX_PKG_DESCRIPTION="Fast and statically typed, compiled language with Ruby-like syntax"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@HertzDevil"
+TERMUX_PKG_VERSION=1.9.0
+TERMUX_PKG_GIT_BRANCH=$TERMUX_PKG_VERSION
+TERMUX_PKG_SRCURL=git+https://github.com/crystal-lang/crystal
+TERMUX_PKG_AUTO_UPDATE=true
+_LLVM_MAJOR_VERSION=$(. $TERMUX_SCRIPTDIR/packages/libllvm/build.sh; echo $LLVM_MAJOR_VERSION)
+_LLVM_MAJOR_VERSION_NEXT=$((_LLVM_MAJOR_VERSION + 1))
+TERMUX_PKG_DEPENDS="libevent, libgc, libgmp, libiconv, libllvm (<< $_LLVM_MAJOR_VERSION_NEXT), libxml2, libyaml, openssl, pcre2, zlib"
```

Comment:
Are `libgmp`, `libiconv`, `libxml2`, `openssl`, `zlib` needed?
Doesn't appear to listed in DT_NEEDED for crystal and shards.

Also can you add `libc++`?

Reply:
These libraries are part of Crystal's standard library.